_posts/2024-07-20-datafusion-comet-0.1.0.md

```diff
@@ -0,0 +1,153 @@
+---
+layout: post
+title: "Apache DataFusion Comet 0.1.0 Release"
+date: "2024-07-20 00:00:00"
+author: pmc
+categories: [subprojects]
+---
+
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+The Apache DataFusion PMC is pleased to announce the first official source release of the Comet subproject.
```

Comment:
I recommend adding a link to the homepage https://datafusion.apache.org/comet/

Something like 


```suggestion
The Apache DataFusion PMC is pleased to announce the first official source release of the [Comet](https://datafusion.apache.org/comet/) subproject.
```

Perhaps